src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -166,8 +189,14 @@ private LibraryExport ExportProject(ProjectDescription project)
                 sourceReferences.Add(sharedFile);
             }
 
+            var contentFiles = project.Project.Files.GetContentFiles().Select(f => new LibraryContentFile()
```

Comment:
This is new. Not sure we should do this as part of the change. We should change shared project content into this however.